mir-module/src/main/resources/META-INF/resources/js/jquery.search-entity.js

```diff
@@ -475,9 +628,15 @@
         return;
 
       var type = null;
+      var type_fallback = null;
```

Comment:
Codacy found an issue: [Identifier 'type_fallback' is not in camel case.](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=6286752)